### DISCUSSION TITLE
Is it possible to install LunarVim without internet connection?

### DISCUSSION THREAD

**Discussion by qiuwei** [LunarVim/LunarVim#3989] [General]
For example, can I simply copy some directories to install LunarVim?

**Comment by LostInTheLogs** [comment]
https://github.com/LunarVim/LunarVim/issues/2122#issuecomment-1382743642

**Comment by qiuwei** [comment]
If you are using the same OS on different machines(say ubuntu 22.04), you can simply rsync(or copy) the following directories/files to the target machine which does not have internet connection.

```
~/.config/lvim
~/.local/bin/lvim
~/.local/share/lunarvim
```

If you are using the same usernames, then you are good to go. Otherwise, you need to grep and replace the username in `~/.config/lvim` and `~/.local/bin/lvim` because the path is hardcoded.

This saves me a lot of time for setting up lunarvim on new machines.
